assets/js/voidsearch.js

```diff
@@ -100,6 +101,28 @@
         if (tooMany) {
             table.append(tooManyNotice());
         }
+
+        if (packagesExactMatch.length >= 1) {
+            $('#voidSearch_result tr:first').after(
+                packagesExactMatch.map((p) => {
+                    let ghSlug = p.name.replace(/-(?:32bit|dbg)$/, "");
+                    p.anchor = "<a href=\"https://github.com/void-linux/void-packages/tree/master/srcpkgs/"
+                        + ghSlug + "\" target=\"_blank\" title=\"View on GitHub\">" + p.name + "</a>";
+                    p.filename_size = typeof(p.filename_size) == 'number' ? formatSize(p.filename_size) : p.filename_size;
```

Comment:
Maybe make this into a function, if it can avoid the repetition? (Note I don't really know JS)

Reply:
Added `transformPackageForPackageCell(p)`